src/utils/dates.js

```diff
@@ -158,13 +180,77 @@ export function week(date) {
 }
 
 export function today() {
-  return dates.startOf(new Date(), 'day')
+  return startOf(new Date(), 'day')
 }
 
 export function yesterday() {
-  return dates.add(dates.startOf(new Date(), 'day'), -1, 'day')
+  return dates.add(startOf(new Date(), 'day'), -1, 'day')
 }
 
 export function tomorrow() {
-  return dates.add(dates.startOf(new Date(), 'day'), 1, 'day')
+  return dates.add(startOf(new Date(), 'day'), 1, 'day')
+}
+
+export function endOf(date, unit, localizer) {
```

Comment:
Hmm I am not really a fan of overriding basically all of the date math library like this. There are too many forks in behavior that may change depending on the localizer and no explicit contract for how these methods should behave.

Reply:
If we think it makes sense to put the logic in the localizer then we should probably design that directly. e.g. expose these methods on the localizer class and remove the helper. I am confident though that's the best approach though